framework/src/modules/chain/logic/transaction_pool.js

```diff
@@ -19,6 +19,7 @@ const {
 	Status: TransactionStatus,
 	TransactionError,
 } = require('@liskhq/lisk-transactions');
+const processTransactionLogic = require('../logic/process_transaction');
```

Comment:
```suggestion
const { composeProcessTransactionSteps } = require('../logic/process_transaction');
```

and then adjust usages below

Reply:
If I do this way I can't `spy` it during the tests.
I know it's not a good reasoning but there is no easy solution =|